autosilk-cli.yaml

```
@@ -0,0 +1,12 @@
+project-name: cfparams-reporting
+pipeline-slug: cfparams
+cluster-name: cfparams
+github-repo: cfparams
+commit-branch: cfparams-autosilk-init
+create-source-security-groups: false
+create-ecr-repos: false
+restricted-accounts: false
+environments:
```

```suggestion
environments: []
```
This might only need to be an empty list -- the CI resources are always deployed regardless of this list, and there isn't any other infrastructure that we need, right?